ioctl/cmd/ws/wsmessage.go

```diff
@@ -37,11 +37,13 @@ type sendMessageRsp struct {
 	MessageID string `json:"messageID"`
 }
 
+type stateLog struct {
+	State   string    `json:"state"`
+	Time    time.Time `json:"time"`
+	Comment string    `json:"comment"`
```

Comment:
rename to `Comment` does not affect other codes?

Reply:
this datatype defines in w3bstream code